pkg/command/command.go

```diff
@@ -17,13 +17,12 @@ import (
 )
 
 const (
-	RelaxedURLRegEx = `(?:\b)((http(s?):\/\/)?(((www\.)?[a-zA-Z0-9\.\-\_]+(\.[a-zA-Z]{2,3})+)|((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))(\/[a-zA-Z0-9\_\-\.\/\?\%\#\&\=]*)?)(?:\s|\b)`
+	RelaxedURLRegEx = `(http|ftp|https):\/\/([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:\/~+#-]*[\w@?^=%&\/~+#-])`
 )
 
 // Command defines the information required for executing a call to any executable
 type Command struct {
 	ErrorCategoryMapping map[string][]string
-	URLReportFileName    string
```

Comment:
not sure if we should remove this option.
We may have multiple commands running. In my opinion we need a way to define the file name from the outside (when calling a command) and not have it hardcoded here.

Reply:
Currently we are trying to provide SLC-29 compliance so it should be hardcoded, but yeah at the level up to the package(using constructor).

Reply:
should we keep the URL URLReportFileName and also pass it from outside (with a fixed name) ?

Reply:
It outdated code. Currently I've renamed it to the StepName and I use it as a structure field for the log.